ios/FluentUI/Extensions/UIColor+Extensions.swift

```diff
@@ -83,7 +83,7 @@ extension UIColor {
         }
     }
 
-    var dynamicColor: DynamicColor? {
+    public var dynamicColor: DynamicColor? {
```

Comment:
Can you move this higher in the file to match the Microsoft Swift Guide?

https://microsoft.github.io/swift-guide/FileOrganization.html

Reply:
I think it might already be in the right place. It is below some internal functions, but they also happen to be inits and I'm not sure we want to break up the inits?